lite-locals.tf

```diff
@@ -63,6 +190,36 @@ locals {
     replicas     = 6
```

Comment:
Why 6?

Reply:
6 was hard coded probably my those who created the opinionated hpcc. I would think that the number of replicas probably depends on the amount of spraying that is being done. My understanding is that none of these exist until you need them (i.e. on demand). So, there is no cost until they are used.